fabric-commands-v0/src/main/java/net/fabricmc/fabric/api/command/CommandSide.java

```diff
@@ -14,9 +14,16 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.impl.command;
+package net.fabricmc.fabric.api.command;
+
+import com.google.common.annotations.Beta;
 
 public enum CommandSide {
+	/**
+	 * Not Implemented Yet!
+	 */
+	@Beta
```

Comment:
use jetbrains annotations

Reply:
we use loom 0.4
and `Beta` does what it does

Reply:
use jetbrains annotations

Reply:
we use loom 0.4
and we've used beta in the past

Reply:
use jetbrains annotations

Reply:
no